django_app/settings.py

```diff
@@ -13,22 +13,32 @@
 import os
 from pathlib import Path
 
+
+from dotenv import load_dotenv 
+
+
+
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+load_dotenv(BASE_DIR/ '.env') 
+
+
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
 
 from django.core.management.utils import get_random_secret_key
 
-SECRET_KEY = os.getenv("DJANGO_SECRET_KEY", get_random_secret_key())
+SECRET_KEY = os.getenv("SECRET_KEY")
+
+CSRF_TRUSTED_ORIGINS = []
 
```

Comment:
How many of these changes are going to break running locally on the laptop?

Is there a way that we can keep it working with fly.io but ALSO easily run on the laptop?

Reply:
yes, since i configured those variables in env file it can word well on local environment too

Reply:
![image](https://user-images.githubusercontent.com/28830125/218973346-e0640960-b9af-43a7-a4b7-6d8778d0bdfc.png)

Reply:
![image](https://user-images.githubusercontent.com/28830125/218973509-81ba9d85-f5e1-4f4f-bf09-53572b32e77c.png)
in local environment

Reply:
Very nice